SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+## Supported Versions
+
+scikit-image is generally backwards compatible with few exceptions, so we
+recommend users to always use the latest version.
+
+We usually don't backport patches or fixes to previous versions. But we may
+release minor versions with patches. In case of particular critical issues
+exceptions may be made.
+
+## Reporting a Vulnerability
+
+If you believe you've discovered a serious vulnerability, please contact the
+scikit-image core team at: <br>
+skimage-core@discuss.scientific-python.org
+
+We will evaluate your report and if necessary issue a fix and an advisory. If
+the issue was previously undisclosed, we'll also mention your name in the
+release notes.
+
+## Responsible Disclosure
+
+We may apply a responsible disclosure process to reported or otherwise
+discovered vulnerabilities. We will usually do that for a critical
+vulnerability, and only if we have a good reason to believe information about
+it is not yet public.
+
+--
+
+_We are thankful to
+[Redis, from whose SECURITY.md](https://github.com/redis/redis/blob/d27c7413a95a0a271b94376f3ec64dae06326b29/SECURITY.md)
+we drew content and inspiration._
```

Comment:
Not sure if this is enough recognition or if we should also add a copyright to our license file. I basically copied what we do in our CoC.